serve_hello_private.py

```diff
@@ -1,6 +1,8 @@
 from fastapi import FastAPI
 from ray import serve
 
+msg = os.getenv("MSG", "Hello world!")
```

Comment:
Let's name this something more evocative ("SERVE_RESPONSE_MESSAGE")?

Also, let's add a comment about what this does